domain/src/test/java/se/cygni/game/transformation/DecrementStunTest.java

```diff
@@ -33,14 +38,38 @@ public void testTransform() {
     public void testTransformWithZeroCount() {
         CharacterImpl paintbotA = new CharacterImpl("a", "a", 2);
 
-        Tile[] tiles = new WorldState(3, 3).getTiles();
+        WorldState ws = new WorldState(3, 3);
+        Tile[] tiles = ws.getTiles();
         tiles[2] = new Tile(paintbotA);
-        WorldState worldState = new WorldState(3, 3, tiles);
+        WorldState worldState = ws.withTiles(tiles);
 
         DecrementStun decrementStun = new DecrementStun();
 
         WorldState updatedWorldState = decrementStun.transform(worldState);
         CharacterImpl updatedCharacter = updatedWorldState.getCharacterById("a");
         assertEquals(0, updatedCharacter.getIsStunnedForTicks());
     }
+
+    @Test
+    public void testTransformWithoutTouchingCollisionsOrExplosions() {
+        CharacterImpl paintbotA = new CharacterImpl("a", "a", 2);
```

Comment:
Minor: `paintbotA` is never used here, right?

Reply:
No, it's not 😞 Would you like me to remove it?

Reply:
If you want your PR to be top-notch 👌🏻  But it's not necessary :)

Reply:
Everything should always be top-notch 👌